tools/c7n_mailer/c7n_mailer/cli.py

```diff
@@ -207,6 +213,9 @@ def main():
 
         if is_azure_cloud(mailer_config):
             azure_deploy.provision(mailer_config)
+        elif is_gcp_cloud(mailer_config):
+            print('Deploying mailer as a google cloud function is not supported at this time.')
```

Comment:
raise something here so it's not just lost in the logs

Reply:
good advice. Fixed by adding in raise Warning in next commit.